Source/Library/GameBoardWindow/GameBoardPanel.cpp

```diff
@@ -0,0 +1,45 @@
+#include <QGridLayout>
+#include "TicTacToe/Library/GameBoardWindow/GameBoardPanel.hpp"
+#include "TicTacToe/Library/GameBoardWindow/TokenTile.hpp"
+
+using namespace boost;
+using namespace boost::signals2;
+using namespace TicTacToe;
+
+GameBoardPanel::GameBoardPanel(const Board& board, QWidget* parent) :
+  QWidget(parent),
+  m_board(&board) {
+  setFixedSize(304, 304);
+  setStyleSheet("background-color: #FFFFFF;");
+  auto panel_layout = new QGridLayout();
```

Comment:
new QGridLayout(this);

Ownership in Qt is difficult to keep track of, so generally you explicitly indicate the parent when constructing an object.